pkg/bazel/bazel_flags.go

```diff
@@ -64,6 +71,24 @@ var (
 		"start_app":                                       {},
 	}
 
+	// List of all commands with label as inputs. To compile the list, you can
+	// use the following command:
+	//
+	//   bazel help completion | grep 'ARGUMENT="label"'
```

Comment:
Looks like there are `label-test` and `label-bin` variants so the correct grep is `| grep 'ARGUMENT="label'` (without the 2nd double quote) to catch the `test`, `coverage` and `run` commands.

I'll push a commit to update the list

Reply:
```
$ bazel help completion | grep 'ARGUMENT="label'
BAZEL_COMMAND_AQUERY_ARGUMENT="label"
BAZEL_COMMAND_BUILD_ARGUMENT="label"
BAZEL_COMMAND_COVERAGE_ARGUMENT="label-test"
BAZEL_COMMAND_CQUERY_ARGUMENT="label"
BAZEL_COMMAND_FETCH_ARGUMENT="label"
BAZEL_COMMAND_MOBILE_INSTALL_ARGUMENT="label"
BAZEL_COMMAND_PRINT_ACTION_ARGUMENT="label"
BAZEL_COMMAND_QUERY_ARGUMENT="label"
BAZEL_COMMAND_RUN_ARGUMENT="label-bin"
BAZEL_COMMAND_TEST_ARGUMENT="label-test"
```

Reply:
Done.